app/scheduler.py

```diff
@@ -123,14 +124,17 @@ def monthly_remainder():
                     
                     kpi_id = doc['kpi_id']
                     slack_id = doc['slack_id']
+                    email = doc['email']
+                    print(email)
+                    print(month)
                     
                     today_date = int(today.strftime("%d"))
                     #check if user allow date(joiningdate + 10) is greater then to today date then send normal slack msg else create a report with default ratings
-                    if today_date<11:
+                    if today_date<27:
                         if role != 'Admin':
-                            monthly_mesg=mesg.replace("Slack_id:", "<@" + slack_id + ">!")
-                            msg=monthly_mesg.replace(":Month",""+str(month)+"")
-                            slack_message(msg=msg)
+                            monthly_reminder_payload = {"user":{"email":email},"data":month,"message_key":"monthly_reminder","message_type":"simple_message"}
+                            slack_message = requests.post(url=nt_URl,json=monthly_reminder_payload)
```

Comment:
change this to notification_message everywhere

Reply:
ok sir will change this to notification_message